### DISCUSSION TITLE
404 - Wrong url in local environment

### DISCUSSION THREAD

**Discussion by maartenheideman** [trendyminds/craft-palette#41] [Q&A] [answered]
Same issue as in https://github.com/trendyminds/craft-palette/discussions/27 the url for actions links to "http://localhost/actions/palette/actions" and should link to the siteUrl setup in the env file? Looks like the config/palette.php is ignored.

**Comment by aaronbushnell** [comment, marked as answer]
Fixed in 4.0.5, thanks!

**Reply by maartenheideman** [reply]
Great update by the way 👍

**Reply by aaronbushnell** [reply]
Thanks so much, @maartenheideman, that's very kind of you! ❤️
